src/v2/System/Router/RouterLink.tsx

```diff
@@ -56,11 +56,28 @@ type RouterLinkMixinProps = BoxProps & {
   textDecoration?: ResponsiveValue<string>
 }
 
+const VALID_FOUND_LINK_PROPS = [
+  "activeClassName",
```

Comment:
I guess I previously broke active highlighting when I enabled the prop filtering here.